docs/references/config/genesis.json.md

```diff
@@ -0,0 +1,217 @@
+---
+order: 1
+parent:
+  title: genesis.json
+  description: The network genesis file
+  order: 1
+---
+# genesis.json
+On first start, the network parameters are read from the `genesis.json` file into a freshly created database.
+
```

Comment:
We may want to state here that all nodes in a chain network must have _exactly_ the same contents in their `genesis.json` file.

Reply:
Addressed